app/src/command.rs

```diff
@@ -31,6 +31,10 @@ use {
     umi_state::State,
 };
 
+#[cfg(feature = "op-upgrade")]
+const L2_TO_L1_MESSAGE_PASSER_ADDRESS: alloy::primitives::Address =
```

Comment:
You also have this constant defined in `app/src/lib.rs`, it could be imported from there instead of duplicated.